src/app/pages/CreateConnectorPage/components/SearchFilter.tsx

```diff
@@ -0,0 +1,89 @@
+import { validateSearchField } from '@utils/shared';
+import React, { FunctionComponent, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  Button,
+  ButtonVariant,
+  InputGroup,
+  TextInput,
+  TextInputProps,
+  Tooltip,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+
+export type FilterProps = {
+  filterSelected?: string;
+  placeholder: string;
+  onChangeSearchField: (value: string) => void;
+  SearchFieldName: string;
+};
+
+export const SearchFilter: FunctionComponent<FilterProps> = ({
```

Comment:
This is great!  I think since this component is generic enough I'd suggest it go in `app/components`.  

Initially when I started creating these components within the pages directories, it was because they were still just used in that one spot.  I figured once a component is used for a couple different pages like this one will be would be a good time to promote that component to the `app/components` directory.